ui/next/config.js

```diff
@@ -137,6 +139,9 @@ System.config({
     "github:jspm/nodelibs-zlib@0.1.0": {
       "browserify-zlib": "npm:browserify-zlib@0.1.4"
     },
+    "npm:ByteBuffer@0.2.6": {
```

Comment:
that appears to be a different library than npm:bytebuffer, which is what you want. See line 66 in this file.